### PR TITLE
market configs

### PR DESCRIPTION
**Pull request by foodaka**

### CODE REVIEW

File: src/ui-config/marketsConfig.tsx [hunk 1]
```diff
@@ -101,6 +102,26 @@ export const marketsData: {
...
+    marketTitle: 'Ethereum Ropsten',
+    chainId: ChainId.ropsten,
+    enabledFeatures: {
+      faucet: true,
+      governance: true,
+      staking: true,
+      incentives: true,
+    },
+    rpcOnly: true,
+    addresses: {
```
Comment (foodaka) [inline src/ui-config/marketsConfig.tsx:115]: @kartojal can u have a quick look and let me know if these look correct

**Comment by kartojal** [comment]
would be cool to do some (manual) testing of all assets, but ipfs gateway is not loading for me

**Comment by foodaka** [comment]
@bojanaave can you test all the assets on ropsten

**Comment by bojanaave** [comment]
HI @foodaka  after performing smoke testing on the Ropsten network I did not spot any issue.

**Review by kartojal — APPROVED** [review]

**Pull request merged**